railties/test/generators/app_generator_test.rb

```diff
@@ -955,16 +955,20 @@ def test_skip_webpack_install
     end
 
     generator([destination_root], skip_webpack_install: true).stub(:rails_command, command_check) do
-      quietly { generator.invoke_all }
+      generator.stub :bundle_command, nil do
+        quietly { generator.invoke_all }
+      end
     end
 
     assert_gem "webpacker"
     assert_no_file "config/webpacker.yml"
 
-    output = Dir.chdir(destination_root) do
-      `rails --help`
+    output = capture(:stdout) do
+      Dir.chdir(destination_root) { system "bin/rails help" }
     end
+
     assert_match(/The most common rails commands are:/, output)
+    assert_match(/webpacker:install/, output)
```

Comment:
@eugeneius I have added this assertion in, as it was missing from the current test. 

With a systems call, we get the correct output, i.e. all rake tasks including assets and webpacker appear correctly:

```
"The most common rails commands are:\n generate     Generate new code (short-cut alias: \"g\")\n console      Start the Rails console (short-cut alias: \"c\")\n server       Start the Rails server (short-cut alias: \"s\")\n test         Run tests except system tests (short-cut alias: \"t\")\n test:system  Run system tests\n dbconsole    Start a console for the database specified in config/database.yml\n              (short-cut alias: \"db\")\n\n new          Create a new Rails application. \"rails new my_app\" creates a\n              new application called MyApp in \"./my_app\"\n\n\nAll commands can be run with -h (or --help) for more information.\nIn addition to those commands, there are:\n\n  about\n  action_mailbox:ingress:exim\n  action_mailbox:ingress:postfix\n  action_mailbox:ingress:qmail\n  action_mailbox:install\n  action_mailbox:install:migrations\n  action_text:install\n  action_text:install:migrations\n  active_storage:install\n  app:template\n  app:update\n  assets:clean[keep]\n  assets:clobber\n  assets:environment\n  assets:precompile\n  cache_digests:dependencies\n  cache_digests:nested_dependencies\n  credentials:diff\n  credentials:edit\n  credentials:show\n  db:create\n  db:drop\n  db:environment:set\n  db:fixtures:load\n  db:migrate\n  db:migrate:down\n  db:migrate:redo\n  db:migrate:status\n  db:migrate:up\n  db:prepare\n  db:reset\n  db:rollback\n  db:schema:cache:clear\n  db:schema:cache:dump\n  db:schema:dump\n  db:schema:load\n  db:seed\n  db:seed:replant\n  db:setup\n  db:structure:dump\n  db:structure:load\n  db:system:change\n  db:version\n  destroy\n  dev:cache\n  encrypted:edit\n  encrypted:show\n  initializers\n  log:clear\n  middleware\n  notes\n  restart\n  routes\n  runner\n  secret\n  secrets:edit\n  secrets:setup\n  secrets:show\n  stats\n  test:all\n  test:db\n  time:zones[country_or_offset]\n  tmp:clear\n  tmp:create\n  version\n  webpacker\n  webpacker:binstubs\n  webpacker:check_binstubs\n  webpacker:check_node\n  webpacker:check_yarn\n  webpacker:clean[keep,age]\n  webpacker:clobber\n  webpacker:compile\n  webpacker:info\n  webpacker:install\n  webpacker:install:angular\n  webpacker:install:coffee\n  webpacker:install:elm\n  webpacker:install:erb\n  webpacker:install:react\n  webpacker:install:stimulus\n  webpacker:install:svelte\n  webpacker:install:typescript\n  webpacker:install:vue\n  webpacker:verify_install\n  webpacker:yarn_install\n  yarn:install\n  zeitwerk:check\n"
```

We can also make the test pass without this assertion by:

`capture(:stdout) { Rails::Command.invoke("--help") }`

However this does not output the webpacker rake command.

Reply:
If we can remove that system call, we can bring this test down to ~ 0.5s, currently its:

`AppGeneratorTest#test_skip_webpack_install = 4.86 s = .`